aws/lambdas/LogsToElasticSearch_info/index.js

```diff
@@ -0,0 +1,272 @@
+/* eslint-disable */
+
+/*
+/* This Lambda is taken from the magically-generated lambda that is created 
+/* in AWS Console when setting up a log subscription to our Elasticsearch Kibana domain.
+/* We need to host this lambda in order to manage it via Terraform.
+*/
+
+// v1.1.2
+var https = require('https');
+var zlib = require('zlib');
+var crypto = require('crypto');
+
+const ENV = process.env;
+
+var endpoint = ENV.es_endpoint;
```

Comment:
[non-blocking comment, take it or leave it] 

Below we’re referencing `process.env` directly, but here it’s assigned to `ENV` and accessed there. Typically in nodejs, I’d expect it to be referenced with `process.env.ES_ENDPOINT`.

Reply:
Thanks, this doesn't seem necessary - changed! https://github.com/ustaxcourt/ef-cms/pull/387/commits/b6d2d97e80083e6e5196a332b1b35eaf76fd7955